Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -15,6 +15,8 @@ class OverviewViewController: UIViewController {
     @IBOutlet weak var fetchingBalanceIndicator: UIImageView!
     @IBOutlet weak var totalBalanceLabel: UILabel!
     @IBOutlet weak var recentActivityTableView: UITableView!
+    @IBOutlet weak var backupWallet: UIView!
```

Comment:
I think would be enough for this. It will save you from having to do all this and worrying about gesture recognizers.

Reply:
Using a button there won't align the arrow and text properly on all screens